controller/db/db.go

```diff
@@ -42,3 +44,12 @@ func Open(path string) (boltz.Db, error) {
 
 	return db, nil
 }
+
+func LoadCurrentMetadat(tx *bbolt.Tx, name string) uint64 {
```

Comment:
Is the metadata just the index?

Also, `Metadat` to `Metadata` ?

Reply:
Fixed. I'll switch it to GetCurrentMetadata when there are other consumers. There is other stuff stored under metadata, but that's currently the only value that's retrieved with that method.